service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageLegalRepresentativesController.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import io.swagger.annotations.Api;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.events.LegalRepresentativesChangeToCaseEvent;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.service.LegalRepresentativeService;
+import uk.gov.hmcts.reform.fpl.service.validators.ManageLegalRepresentativesValidator;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+
+@Api
+@RestController
+@RequestMapping("/callback/manage-legal-representatives")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ManageLegalRepresentativesController extends CallbackController {
+    private final LegalRepresentativeService legaRepresentativeService;
+    private final ManageLegalRepresentativesValidator manageLegalRepresentativesValidator;
+
+    @PostMapping("/about-to-start")
+    public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackRequest) {
+        return respond(callbackRequest.getCaseDetails());
+    }
```

Comment:
about-to-start is optional, if you do nothing here then method can be removed and callback can be removed from definition

Reply:
removed